core/kazoo_translator-1.0.0/src/convertors/kzt_kazoo.erl

```diff
@@ -65,4 +65,5 @@ req_params(Call) ->
        ,{<<"Transcription-Text">>, kzt_util:get_transcription_text(Call)}
        ,{<<"Transcription-Status">>, kzt_util:get_transcription_status(Call)}
        ,{<<"Transcription-Url">>, kzt_util:get_transcription_url(Call)}
+       | cf_kvs_set:kvs_to_props(Call)
```

Comment:
We're trying hard to not have cross-app dependencies like this. If this is "generic" functionality, it should be moved to a lib in core.